projects/packages/masterbar/src/masterbar/class-masterbar.php

```diff
@@ -135,13 +135,6 @@ public function __construct() {
 		if ( isset( $this->user_data['use_wp_admin_links'] ) ) {
 			update_user_option( $this->user_id, 'jetpack_admin_menu_link_destination', $this->user_data['use_wp_admin_links'] ? '1' : '0' );
 		}
-		// If Atomic, store and install user locale.
-		if ( $this->site_woa && 'wp-admin' !== get_option( 'wpcom_admin_interface' ) ) {
-			$this->user_locale = $this->get_jetpack_locale( $this->user_locale );
-			$this->install_locale( $this->user_locale );
```

Comment:
Can we also remove the unused helper methods?

Reply:
I'm a bit afraid of removing these methods since it's referred by the deprecated Masterbar module: https://github.com/Automattic/jetpack/blob/92d2878741213a9dc4ad547d38a1ab105687ca6c/projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php#L243.
I believe we should best wait for the removal of the Masterbar: https://github.com/Automattic/jetpack/issues/35469

Reply:
Ohhh......... I see! Thanks for the context.